### ISSUE TITLE
Add login and authentication system

### ISSUE THREAD

**Issue by 6510615138** [6510615138/register#3]
Allow user to log in via password

**Comment by Hamidusman** [comment]
Hello there, i would like to work on this

**Comment by 6510615138** [comment]
Hi there, this is my school project, a class registration system, and it is my very first project working with Django.
Didn't expect anyone to discover this project and I'm very excited to work with you! 

Oh, about the project, I already made login by username, any improvement is welcome, I think maybe we add other to login  such as email would be great.

**Comment by Hamidusman** [comment]
So far as I can be of help I don't mind.

I'll see what i can do

**Comment by Hamidusman** [comment]
I've been trying to run the server but i keep getting this error.

-  cannot import name 'load_env' from 'loadenv'

despite the fact that I already installed it. Just thought i should ask u as well

**Comment by 6510615138** [comment]
yesterday I got a notice from a friend that I put secret key in main branch which isn't a good practice and I shouldn't overlook this issue  so I made a fix, that might not work properly. 

In register/settings.py 
delete line 14,24
 - from loadenv import load_env
 - env = load_env('.env')

line 25 change SECRET_KET to some random string 
 - SECRET_KEY = env["GOLDEN_APPLE"]

and everything should work just fine.

**Comment by Hamidusman** [comment]
This error came up after the last:

- django.core.exceptions.ImproperlyConfigured: WSGI application 'register.wsgi.application' could not be loaded; Error importing module.

Did you make any changes to WSGI?

**Comment by 6510615138** [comment]
No, I didn't make any changes to WSGI. I think the problem is path in settings.py
I just fix it by update path and it should work now, please check it out.
